ffi/java-idsdk/README.md

```diff
@@ -0,0 +1,69 @@
+# Java wrapper for the ID SDK
+
+FFI wrapper using JNA
+
+## Build
+
+`mvn pakage`
```

Comment:
We should check it works on OSX & Win64 (or have you already tried, @mihai-dinculescu ?)